public/blog/star-history-yearly-pick-2022-data-infra-tools.md

```diff
@@ -0,0 +1,146 @@
+
+OPA's policies are expressed using Rego, a high-level declarative language. Rego makes it easy to define policies and configure policies (Policy-as-Code) as if you were writing code. For example, questions like: Is Bob allowed to make GET requests to `/api/v1/products`? What records does he have permission to view?
+
+![opa-workflow](/blog/assets/yearly-pick-data-infra-tools-2022/opa-workflow.webp)
+
+OPA's [Adopter.md](https://github.com/open-policy-agent/opa/blob/main/ADOPTERS.md) includes a number of big names using OPA in prod: Netflix, Pinterest, T-Mobile, etc. The syntax for policies has yet to be standardized, leading to everyone having to build the same wheels repeatedly, and OPA has the potential to change that.
+
+## Steampipe
+
+![steampipe](/blog/assets/yearly-pick-data-infra-tools-2022/steampipe.webp)
+
+With [Steampipe](https://github.com/turbot/steampipe), you can use SQL to instantly query your cloud services (AWS, Azure, GCP, and more). [Steampipe Cloud](https://steampipe.io/docs/cloud/overview), a fully managed SaaS platform for hosting Steampipe instances, was released January, 2022.
+
+Steampipe's star skyrocketed in October, 2022, thanks to landing on the [Hacker News](https://news.ycombinator.com/item?id=33034351) front page.
+
+![steampipe-hn](/blog/assets/yearly-pick-data-infra-tools-2022/steampipe-hn.webp)
+
+## Tier
+
+![tier](/blog/assets/yearly-pick-data-infra-tools-2022/tier.webp)
+
+[Tier](https://github.com/tierrun/tier) is a tool to manage pricing and billing workflow for your SaaS offering
+
+Tier landed on [Hacker News front page](https://news.ycombinator.com/item?id=33429972), and their stars skyrocketed, just like Steampipe.
+
+![tier-hn](/blog/assets/yearly-pick-data-infra-tools-2022/tier-hn.webp)
+
+Pricing SaaS products is a constant WIP, but it is a complex task and is often tied up in complex workflows and hidden cross-organizational dependencies. Many startups choose a pricing strategy for their product once, which stays the same forever.
+
+Tier has a top-level design for how to implement pricing that allows for changes to pricing models without changing application code or business processes (Pricing as Code). The founding team thoroughly thought about pricing and built a website explaining t[he 5 Pillars of PriceOps](https://priceops.org/).
+
+![tier-priceops](/blog/assets/yearly-pick-data-infra-tools-2022/tier-priceops.webp)
+
+This time around, Star History is highlighting some relatively lesser-known projects that are on the rise. We see that the **Data & Infra & Tools** section can be divided into two main categories:
```

Comment:
```suggestion
For Star History, our goal is to discover interesting and inspiring projects. The aforementioned projects are not household names in the open source community (yet), but each has great potential since they are tackling genuine issues with novel ideas. We see that the **Data & Infra & DevTools** section can be divided into two main categories:
```